src/com/google/enterprise/adaptor/Config.java

```diff
@@ -1014,7 +1014,7 @@ File getConfigFile() {
    * @return map of all endings of prefix split into key value pairs
    */
   public synchronized Map<String, String> getValuesWithPrefix(String prefix) {
-    Map<String, String> values = new HashMap<String, String>();
+    Map<String, String> values = new TreeMap<String, String>();
```

Comment:
done. i am not sure if beneficial or not. i was thinking to proactively avoid sort problem.  in general i try to use TreeMaps rather than HashMaps for Maps that are typically less than several hundred keys and also for tests. sometimes i forget and type HashMap out of  habit. anyway: done.